src/ResizableBox.tsx

```diff
@@ -120,9 +124,11 @@ export const ResizableBox: React.FC<ResizableBoxProps> = ({
     // into multiple lines
     <Box
       {...props}
-      {...(!disabled && {
-        width: `clamp(35rem, calc(35rem + ${widthDelta}px), 70vw)`,
-      })}
+      style={
+        !disabled
+          ? { width: `clamp(35rem, calc(35rem + ${widthDelta}px), 70vw)` }
+          : undefined
```

Comment:
this is a performance optimization - such dynamic values like this one should not be "pumped" through Emotion because each intermediate value will result in a separate class name being created and injected in the CSSOM. Using inline style for this is strictly better here